provider/app.py

```diff
@@ -19,25 +19,35 @@
 from consumer import Consumer
 from database import Database
 from urlparse import urlparse
+import requests
+
 
 app = Flask(__name__)
 database = Database()
-
 consumers = dict()
 
-
 @app.route('/triggers/<namespace>/<trigger>', methods=['PUT'])
 def postTrigger(namespace, trigger):
     body = request.get_json(force=True, silent=True)
-
     triggerFQN = '/' + namespace + '/' + trigger
-    logging.info("About to create consumer with " +
-                 triggerFQN + " " + str(body))
-    createAndRunConsumer(triggerFQN, body)
-    logging.info("Finished creating consumer.")
-    result = {'success': True}
-    return jsonify(result)
+    logging.info("Ensuring user has access rights to post a trigger")
```

Comment:
Let's make a habit of prefixing all logging messages with `[triggerFQN]` wherever possible. This will add context in a way that is parseable and searchable.

```python
logging.info("[{}] Ensuring user has access rights to post a trigger".format(triggerFQN))
```